app/services/album.js

```diff
@@ -0,0 +1,34 @@
+const fetch = require('node-fetch');
```

Comment:
why did you choose node fetch instead of other package like request or axios?

Reply:
It is the one I have use in the other training. Shout I use other?

Reply:
Check all the 3 packages stars, issues and last PRs on github, and then tell me which one you choose :)

Reply:
I will use axios, request don't have promise handle directly and axios have much more stars and movement than the others.